receiver.go

```diff
@@ -87,6 +97,13 @@ func (r *receiver) parseLoop() {
 	r.frameDecoder = nil
 }
 
+func (r *receiver) fatalError(err error) {
+	r.context.log("Error: parseLoop closing socket due to error: %v", err)
+	r.parseError = err
+	r.conn.Close()
+	//TODO: Should set a WebSocket close code/msg, but websocket.Conn has no API for that
```

Comment:
Can you write another comment about the implications of not sending a Websocket close code/msg?  Will this cause any failures?  Also, might be worth mentioning that Gorilla/websocket has this feature.

Reply:
It just means that the peer won't know why the connection was closed. CBL treats even a normal closure by the server as an error, because the server's not supposed to close the connection. So this is basically harmless.